ops/db-connect.sh

```diff
@@ -1,30 +1,87 @@
-#!/bin/bash
-
-# Call this to connect to cloud Postgres for a specific Data Repo instance.
-# For example, to connect to the dev datarepo database as user mm, run:
-# $ DB=datarepo SUFFIX=mm ENVIRONMENT=dev ./db-connect.sh
-# To connect to a user instance's stairway database, run:
-# $ DB=stairway-mm SUFFIX=mm ENVIRONMENT=dev ./db-connect.sh
+#!/bin/sh
+#
+# This script is used to connect to the TDR Cloud SQL database in a GCP project.
+#
+# It uses the gcloud CLI to get the credentials for the cluster, kubectl to port
+# forward to the SQL proxy pod, and psql to connect to the database.
+#
+# You MUST have gcloud, kubectl, jq, and psql installed to run this script. In
+# addition, you MUST be connected to the VPN to access the database.
+#
+# Usage: ./db-connect.sh
+#
+# The following environment variables may be set to override the defaults:
 #
-# If the connection times out, check that you are on the Broad VPN and are connected to correct
-# Kubernetes cluster.
+#   PROJECT     the GCP project that contains the Cloud SQL database
```

Comment:
Could this use `env` as before? Using env, it seems like you could generate `project`, `secret` and `database`. Requiring the user to specify each would make it harder to use than before.

Reply:
These are all optional arguments, perhaps I can make that clear.

Reply:
I would likely need a large case statement, especially because the paths in GSM do not follow the same structure as the vault paths. Additionally, the way it's currently implemented actually will not work in the majority of cases (e.g. staging).

Reply:
I want to ditto Phil that this change has me confused on how I'd run this script.  I'd prefer some added complexity within the script itself by way of a case statement or helper methods that extrapolate those variables from an ENV, rather than delegating that work to the caller.